src/App.tsx

```diff
@@ -1,25 +1,8 @@
 import * as React from 'react';
-import {createBrowserRouter, RouterProvider} from 'react-router-dom';
-import TeamOverview from './pages/TeamOverview';
-import Teams from './pages/Teams';
-import UserOverview from './pages/UserOverview';
+import Router from 'router';
 
 const App = () => {
-    var router = createBrowserRouter([
-        {
```

Comment:
Keeping the routing configuration in a separate file helps maintain a clear and focused structure in the main App.tsx file. It separates concerns and allows each file to have a single responsibility.